db/20160722_seeds.rb

```diff
@@ -4,6 +4,9 @@ module WorkForwardNola
 
   Sequel.seed do
     def run
+
+      Trait.set_allowed_columns :name
```

Comment:
I tried adding this to fix the build issue, but I don't think it helped. Just changed the message/error location.